.github/scripts/setup_spark_remote.sh

```diff
@@ -2,31 +2,36 @@
 
 set -xv
 
-mkdir -p $HOME/spark
-cd $HOME/spark
+mkdir -p "$HOME"/spark
+cd "$HOME"/spark || exit
```

Comment:
Exit with non-zero status if cd fails. Like exit 1. Same for other places.

Reply:
done